remake.yml

```diff
@@ -15,20 +15,17 @@ targets:
   all:
     depends: 3_visualize/out/figure_1.png
 
-  site_data:
-    command: download_nwis_data()
+  download_files:
```

Comment:
Another approach is to create a file for each target, which would be 

```yaml
1_fetch/out/nwis_01427207_data.csv:
   command: download_nwis_site_data(target_name, parameterCd = I('00010'))
1_fetch/out/nwis_01432160_data.csv:
   command: download_nwis_site_data(target_name, parameterCd = I('00010'))
```
But I know we discussed this gets nasty to copy/paste.

I tried to make the downloads fail _just enough_ to make this seem like a good idea (to split them up)